HumanControls/HumanControls.ino

```diff
@@ -36,17 +40,35 @@ MenuController menuController(ENCODER_PIN_CLK, ENCODER_PIN_DT, DISPLAY_ADDRESS,
                               ANGLE_INCREMENT, ANGLE_MIN, ANGLE_MAX, PRESSURE_INCREMENT, PRESSURE_MIN, PRESSURE_MAX,
                               DURATION_INCREMENT, DURATION_MIN, DURATION_MAX);
 
+EnableController enableController;
+FireController fireController;
+
+bool isConnected = false;
+String statuses[3] = {"disabled", "enabled", "primed"};
```

Comment:
It would probably be better to make this an enum, and then make a function that returns `const char *` with hard-coded strings.

Reply:
We decided against enums, and made statuses all static const char*.